SparkPost/matchbox
# UX-773 Fix color function

### What Changed
- Fixes the broken `color` scss function
- Adds a test page for this so we'll know in the future if the function breaks
- Adds the correct `files` in design-token's `package.json`
- Upgrades libby

### How To Test or Verify
- Run `npm i` locally
- Visit: http://localhost:9001/?path=SCSS-Tokens__Color-Function
- Make sure the page renders correctly



### PR Checklist
